src/components/institution/SelectInstitutionForm.tsx

```diff
@@ -157,6 +166,7 @@ export const SelectInstitutionForm = ({
                   {({ field }: FieldProps<Organization>) => (
                     <>
                       <Autocomplete
+                        value={field.value}
```

Comment:
This is to ensure that when the view opens it shows the selected subunit both in the list and in the search field